src/vcpkg-test/plan.cpp

```diff
@@ -58,6 +58,34 @@ static void remove_plan_check(RemovePlanAction& plan, std::string pkg_name, Trip
     REQUIRE(pkg_name == plan.spec.name());
 }
 
+static ActionPlan create_feature_install_plan(const PortFileProvider& port_provider,
+                                              const CMakeVars::CMakeVarProvider& var_provider,
+                                              View<FullPackageSpec> specs,
+                                              const StatusParagraphs& status_db)
+{
+    const CreateInstallPlanOptions create_options{Test::X64_ANDROID, "pkg"};
+    return create_feature_install_plan(port_provider, var_provider, specs, status_db, create_options);
+}
+
+static ActionPlan create_feature_install_plan(const PortFileProvider& port_provider,
+                                              const CMakeVars::CMakeVarProvider& var_provider,
+                                              View<FullPackageSpec> specs,
+                                              const StatusParagraphs& status_db,
+                                              Triplet host_triplet)
+{
+    const CreateInstallPlanOptions create_options{host_triplet, "pkg"};
+    return create_feature_install_plan(port_provider, var_provider, specs, status_db, create_options);
+}
+
+static ActionPlan create_upgrade_plan(const PortFileProvider& provider,
+                                      const CMakeVars::CMakeVarProvider& var_provider,
+                                      const std::vector<PackageSpec>& specs,
```

Comment:
I would prefer a function that does `std::vector<PackageSpec>` -> `View<FullPackageSpec>` to avoid needing to consider multiple conversions at once, but not a huge deal.

Reply:
I'm not sure what you mean by this. Which multiple conversions are we talking about?

Reply:
Hmm OK I guess there is only one conversion here; I got tripped up because parameters 0, 1, and 3 are the same, but parameter 2 differs, and I think I read it as parameter 3 being either StatusParagraphs or Triplet.

It is still annoying to need to compare conversions to `View<FullPackageSpec>` and `const std::vector<PackageSpec>&`, but you have already resolved that problem by renaming the latter so we good.